app/scripts/background.js

```diff
@@ -87,7 +88,15 @@ const BADGE_MAX_COUNT = 9;
 
 // Setup global hook for improved Sentry state snapshots during initialization
 const inTest = process.env.IN_TEST;
-const localStore = inTest ? new ReadOnlyNetworkStore() : new LocalStore();
+const migrator = new Migrator({
+  migrations,
+  defaultVersion: process.env.WITH_STATE
+    ? FIXTURE_STATE_METADATA_VERSION
+    : null,
+});
+
+const _localStore = inTest ? new ReadOnlyNetworkStore() : new ExtensionStore();
+const localStore = new PersistanceManager({ localStore: _localStore });
```

Comment:
Nit: Rather than use the variable names `_localStore` and `localStore`, maybe `localStore` and `persistenceManager` would be more accurate?

It's a bit confusing to use the term "local store" for both. And this goes against our naming conventions (underscore usually denotes an unused parameter, but this one is used).

Reply:
Addressed in 261351e882

Reply:
Hmm, why `lStore` though? That is also against our "no abbreviations" naming convention.

Reply:
oops, that was not intentional

Reply:
updated name in f4356d8669